jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -103,6 +103,7 @@ class LineAnalysis(PluginTemplateMixin, DatasetSelectMixin, SpectralSubsetSelect
     results = List().tag(sync=True)
     results_centroid = Float().tag(sync=True)  # stored in AA units
     line_items = List([]).tag(sync=True)
+    line_menu_items = List([{}]).tag(sync=True)
```

Comment:
do we need both of these for any reason?  Or can we just make `line_items` the dictionary that you need now to separate the title and value?  Otherwise I'm worried they could fall out of sync and introduce extra overhead.

Reply:
Eh, I don't think that would save any complexity, since we need the index of the `line_items` list based on `name_rest` elsewhere to figure out which `line_mark` to use and that would get more complicated if we had to parse the list of dictionaries. They only each get updated in the same function and nowhere else, so I'm not worried about them getting out of sync.

Reply:
What about a property that just returns `[item['value'] for item in self.line_menu_items]` for the cases where we need to check if an item is in the list or determine the index?  Otherwise I think this could at least use some inline comments to avoid development mistakes down the road.

Reply:
Alright, I like the property idea rather than updating two things semi-independently, I'll make that change.